src/View/ImageFusion/ManualFusionLoader.py

```diff
@@ -81,13 +101,12 @@ def _load_with_vtk(self, progress_callback):
                 )
                 if progress_callback is not None:
                     progress_callback.emit(("Error loading images", error_msg))
-                    logging.error(error_msg)
+                    logging.error("<manualFusionLoader.py_load_with_vtk: >", error_msg)
```

Comment:
what's with the pointy brackets?  If it's not valid XML, there's not a whole lot of need for that.
If you want logs to be readily consumed by an AI, use jsonl format for logging.